### DISCUSSION TITLE
ursina.physics_engine not available

### DISCUSSION THREAD

**Discussion by lyubomyrk** [pokepetter/ursina#397] [Q&A] [answered]
Hello,

I'm new to GitHub and ursina so please be kind and direct me elsewhere if this isn't the correct place. I'm messing around with colliders, and I wanted to implement gravity. In the urisa API reference at https://www.ursinaengine.org/cheat_sheet.html it has exactly what I needed.

![Screenshot from 2022-04-03 12-25-15](https://user-images.githubusercontent.com/35161893/161444619-a059d961-1e7c-4e2b-989c-f7bbd1ff7697.png)
 
I was confused that it's well documented but not present in the ursina project structure, is this a placeholder?  Also a side question, but I'm also learning panda3d, and since ursina is built on top of it, could I use the panda3d physics engine?

**Comment by pokepetter** [comment, marked as answer]
This has been accidentally included in the documentation. Probably from that one time I accidentally committed all changes and pushed before noticing :)) Will remove.
For gravity you just move the entity down every frame. Depending on the use case you might want to make it fall faster over time too. You can do a collision check to see if it's on the ground or not, with raycast or intersects, and stop it if it is.

**Reply by JamesTheFurryCoder** [reply]
sounds like your going to add a way for us to add physics objects in the game in the future. that would sound so cool!
